### DISCUSSION TITLE
自编译Linux4.9内核ksu，超级用户正常，模块加载正常，但模块不能正常安装

### DISCUSSION THREAD

**Discussion by KNKJ55** [tiann/KernelSU#1306] [Q&A]
编译的Linux内核版本为4.9
以前用别人的内核都正常，也刷了几个模块，现在自己编译的内核运行ksu时超级用户正常，之前用别人编译的内核安装的模块正常加载，但是在管理器模块那里安装新模块在安装过程中就会自动重启，不管哪个模块都一样，重启后模块界面没有刚刚安装的模块
不管是kprobe自动集成还是手动修改源码都会有这个问题
有人知道是什么情况吗
设备: 魅蓝Note6
系统: CrDroid 7.20 Android 11
内核源码: github.com/kj-network/k-nel-m1721

**Comment by KNKJ55** [comment]
换了个内核源码，解决了()

**Reply by p88oo** [reply]
可以请教一下编译流程吗，我想给小米平板5pro的4.19.113内核集成ksu，想参考一下

**Reply by KNKJ55** [reply]
> 可以请教一下编译流程吗，我想给小米平板5pro的4.19.113内核集成ksu，想参考一下

装ubuntu，装依赖，准备工具链，拉源码，集成ksu，最后编译刷入，就这样

**Reply by p88oo** [reply]
好的，谢谢
